presto-docs/src/main/sphinx/functions/json.rst

```diff
@@ -22,12 +23,22 @@ Cast to JSON
         SELECT CAST(ARRAY[1, NULL, 456] AS JSON); -- JSON '[1,null,456]'
         SELECT CAST(ARRAY[ARRAY[1, 23], ARRAY[456]] AS JSON); -- JSON '[[1,23],[456]]'
         SELECT CAST(MAP(ARRAY['k1', 'k2', 'k3'], ARRAY[1, 23, 456]) AS JSON); -- JSON '{"k1":1,"k2":23,"k3":456}'
+        SELECT CAST(MAP(ARRAY['k1', 'k2', 'k3'], ARRAY[1, 23, 456]) AS JSON); -- JSON '{"k1":1,"k2":23,"k3":456}'
+        SELECT CAST( -- JSON '[123,"abc",true]'
+            CAST(ROW(123, 'abc', true) AS ROW(value1 BIGINT, value2 VARCHAR, value3 BOOLEAN))
+            AS JSON);
 
-    Note that casting from NULL to ``JSON`` is not straightforward. Casting
+.. note::
+    Casting from NULL to ``JSON`` is not straightforward. Casting
     from a standalone ``NULL`` will produce a SQL ``NULL`` instead of
     ``JSON 'null'``. However, when casting from arrays or map containing
     ``NULL``\s, the produced ``JSON`` will have ``null``\s in it.
 
+.. note::
+    When casting from ``ROW`` to ``JSON``, Presto chooses to use JSON array
```

Comment:
```
When casting from ``ROW`` to ``JSON``, the result is a JSON array rather
than a JSON object. This is because positions are more important than
names for rows in SQL.